Cargo.toml

```diff
@@ -27,7 +27,7 @@ futures = "0.3.1"
 hex = "0.4"
 ironcore-search-helpers = { version = "0.1.2", optional = true }
 itertools = "0.10"
-jsonwebtoken = "7.2"
+jsonwebtoken = { git = "https://github.com/IronCoreLabs/jsonwebtoken" }
```

Comment:
When we depend on forks we often lock it to a commit. Also, we could consider upgrading jsonwebtoken's dependency on base64 0.13 while we're here, since it's what we depend on.